patternfly/patternfly-elements
# fix: Add a warning about updating the on attribute before upgrade

### What has changed and why
Summarize files edited as part of this MR along with a brief description of what was changed/why.

* Added a warning to be thrown before upgrade if the tag contains the `on` attribute
* Updated documentation and tests, removed current examples of manually pushing the `on` attribute inside the pfe-content-set

### Testing instructions
Be sure to include detailed instructions on how your update can be tested by another developer.

1. Check out the branch. 
2. Edit the `pfe-content-set` demo page to add the `on="light"` attribute on the `pfe-content-set` and `pfe-cta` tags.  Add an ID to one of the tags.
3. Run `npm install && npm run start pfe-accordion`.
4. Preview the accordion's demo page.
3. Check that the console is registering 2 warnings and that the one for the tag with an ID lists the ID in the warning in square brackets:

```
pfe-content-set[#foo]: The "on" attribute is protected and should not be manually added to a component. The base class will manage this value for you on upgrade.
```
```
pfe-cta: The "on" attribute is protected and should not be manually added to a component. The base class will manage this value for you on upgrade.
```

### Ready-for-merge Checklist



**Be sure to share your updates with the [patternfly-elements-contribute@redhat.com](mailto:patternfly-elements-contribute@redhat.com) mailing list!**
